### PR TITLE
IMPORT: Improve wso21 fixture data

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

File: spec/fixtures/files/sac_imports_src/NAV3_fixture.csv [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
-"4200006",1,"QUAL-1",2024-03-10,2030-12-31,2031-01-01,2034-12-31,"SAC Tourenleiter*in 1 Winter"
-"4200007",1,"QUAL-1",2023-06-25,2030-12-31,2031-01-01,2034-12-31,"Murmeltierdoktor*in"
-"4200008",1,"QUAL-1",2022-03-12,2028-12-31,2029-01-01,2032-12-31,"SAC Tourenleiter*in 1 Winter"
-"4200009",1,"QUAL-2",2017-10-14,"NULL","NULL","NULL","SAC Tourenleiter*in Bergwandern"
-"4200010",1,"QUAL-1",2023-06-23,2029-12-31,2030-01-01,2033-12-31,"SAC Tourenleiter*in Alpinwandern"
-"4200000",1,"QUAL-2",2021-06-12,"NULL","NULL","NULL","SAC Tourenleiter*in Bergwandern"
-"4200001",1,"QUAL-1",2021-06-06,2029-12-31,2030-01-01,2033-12-31,"SAC Tourenleiter*in Sportklettern"
-"4200002",1,"QUAL-1",2022-07-30,2029-12-31,2030-01-01,2033-12-31,"SAC Tourenleiter*in 1 Sommer"
-"4200003",1,"QUAL-1",2020-07-03,2028-12-31,2029-01-01,2032-12-31,"SAC Tourenleiter*in 1 Sommer"
-"4200004",1,"QUAL-1",2024-02-04,2030-12-31,2031-01-01,2034-12-31,"SAC Tourenleiter*in 1 Winter"
```
Comment (rnestler) [inline spec/fixtures/files/sac_imports_src/NAV3_fixture.csv:39]: I'm not sure about the number and data normalizations. How are these in the original data?

Comment (openscript) [reply spec/fixtures/files/sac_imports_src/NAV3_fixture.csv:39]: They are not quoted.

Comment (openscript) [reply spec/fixtures/files/sac_imports_src/NAV3_fixture.csv:39]: I'm unsure if dates should be quotes?

Comment (rnestler) [reply spec/fixtures/files/sac_imports_src/NAV3_fixture.csv:39]: Damn in the WSO2 source they are...

Comment (rnestler) [reply spec/fixtures/files/sac_imports_src/NAV3_fixture.csv]: I'll only normalize WSO21 for now, there it survives the roundtrip.

**Review by openscript — APPROVED** [review]
Super. LGTM!

**Pull request merged**